lms/services/canvas_api/schemas.py

```diff
@@ -0,0 +1,53 @@
+from enum import Enum
```

Comment:
Added this file here as I initially thought this were the types being returning but they are only validated against them.

Well clean the ones not used and move them back to `canvas/client`